configurations/admin.template.ini

```diff
@@ -1251,6 +1251,12 @@ moduls.embedShareStream.label = "Enable embed & social stream in EP"
 moduls.embedShareStream.permissionName = FEATURE_EMBED_SHARE_SOCIAL_STREAM
 moduls.embedShareStream.group = GROUP_ENABLE_DISABLE_FEATURES
 
+moduls.uncategorizedEntriesValidation.enabled = true
+moduls.uncategorizedEntriesValidation.permissionType = 2
+moduls.uncategorizedEntriesValidation.label = "Do not auto allow uncategorized entries"
```

Comment:
i think it doesnt fully represent what it does, as it relevant only to entitlements.
i would change it to something like "Disable category entry entitlement check"